improver_tests/calibration/ensemble_calibration/test_EstimateCoefficientsForEnsembleCalibration.py

```diff
@@ -508,6 +484,7 @@ def setUp(self):
         halo surrounding the original data.
         Set up expected results for different situations.
         """
+        pytest.importorskip("statsmodels")
```

Comment:
Thanks @MoseleyS. I think your queries are all related to @tjtg's PR to this PR (https://github.com/gavinevans/improver/pull/13). You're correct that this PR makes statsmodels no longer optional, however, as statsmodels is still only used in one particular place in the codebase, rather than having widespread usage like numpy or Iris, @tjtg has recommended that statsmodels is regarded as an optional module. I think this is a similar set-up to pysteps. Therefore, if someone wants to use components of improver, but isn't interested in EMOS, then they don't need to have statsmodels in their environment.